src/components/List/List.tsx

```diff
@@ -1,18 +1,20 @@
 import * as React from 'react';
-import {ListItem} from 'types';
-import Card from '../Card';
-import {Spinner} from '../Spinner';
-import {Container} from './styles';
+import {ListItem} from 'types/types';
+import Card from '../Card/Card';
+import {Spinner} from '../Spinner/Spinner';
+import {ListContainer} from './List.styled';
 
-interface Props {
-    items?: ListItem[];
+interface ListProps {
+    items: ListItem[];
     hasNavigation?: boolean;
-    isLoading: string;
+    isLoading: boolean;
 }
 
-const List = ({items, hasNavigation = true, isLoading}: Props) => {
+const List = (props: ListProps): JSX.Element => {
```

Comment:
Changed the props from `Props` to `ListProps`, to better specify it, and destructure them on the component. Also, specified the component returning type